<?xml version="1.0"?>
<launch>
    <!--Variables for the robot0-->
    <arg name="robot0_name" value="robot0"/>
    
    <!--Create namespace for robot0 and start everything that belongs in the namespace inside the group-->
    <group ns="$(arg robot0_name)">
        <node pkg="fp_tests" type="fake_pose_publisher.py" name="fake_pose_publisher_node" output="screen">
            <param name="rate" value="10"/>
            <param name="set_pose_topic" value="set_fake_robot_pose"/>
            <param name="cmd_vel_topic" value="mobile_base_controller/cmd_vel"/>
            <param name="robot_pose_topic" value="robot_pose"/>
            <param name="tf_prefix" value="robot0"/>
            <param name="odom_frame_id" value="odom"/>
            <param name="robot_base_frame_id" value="base_footprint"/>
        </node>
    </group>
</launch>